src/main/java/FizzBuzz.java

```diff
@@ -11,6 +11,13 @@ public static void main(String[] args) {
     }
 
     public String fizzBuzz(int number) {
+        if (true) {
+            if (true) {
+                if (false) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.yo1000.com/coding_rules#rule_key=squid%3AS1066)
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.yo1000.com/coding_rules#rule_key=squid%3AS2583)